src/helpers/getOverlayPositionBasedOn.js

```diff
@@ -39,5 +39,5 @@ export default (element, overlay, align = 'left') => {
     right = 'auto';
     left = buttonLeft - (overlayWidth - buttonWidth);
   }
-  return { left, right };
+  return { left: Math.max(left, 0), right };
```

Comment:
Do you need to do the same thing for `right` to make sure it won't be cut off on the right side of the bounding box?

Reply:
I tested this more and found a bug that I fixed

As for `right`, I don't think we need anything. The `right` get set to "auto" or "16" so it doesn't seem to overflow. It's mainly just `left`that get set to a negative number sometime and appear off screen

Reply:
I see, thanks for the clarification!